ignite/templates/module/create/files/base/x/{{moduleName}}/module.go.plush

```diff
@@ -56,12 +56,6 @@ func NewAppModuleBasic(cdc codec.BinaryCodec) AppModuleBasic {
 	return AppModuleBasic{cdc: cdc}
 }
 
-// IsOnePerModuleType implements the depinject.OnePerModuleType interface.
-func (AppModuleBasic) IsOnePerModuleType() {}
-
-// IsAppModule implements the appmodule.AppModule interface.
-func (AppModuleBasic) IsAppModule() {}
```

Comment:
@julienrbrt why are these two not necessary anymore?

Reply:
These are necessary but it is already defined on `AppModule`:
https://github.com/ignite/cli/pull/3676/files#diff-4f41d7484bd863bf89001a6adae276094f9dc2b071ad992cfdb235d5565254b1L177-L181